activities.json

```
@@ -227,6 +227,18 @@
     "title": "Fetch Metadata Request Headers",
     "url": "https://github.com/w3c/webappsec-fetch-metadata"
   },
+  {
+    "ciuName": null,
+    "description": "This proposal gives web applications a way to register their ability to handle (read, stream, edit) files with given MIME types and/or file extensions.",
+    "id": "wicg-file-handling",
+    "mozBugUrl": null,
+    "mozPosition": "defer",
+    "mozPositionDetail": "Not far enough along to properly evaluate.",
+    "mozPositionIssue": 158,
+    "org": "Other",
```

```suggestion
    "org": "W3C",
```
Is this really what we want for WICG? Seems to give it more legitimacy than it really has.

---

Drive-by +1 to distinguish CG/BG documents from WG documents. They are still part of the W3C, so "Other" doesn't really fit either.

---

For consistency with existing practice this particular commit should probably be `"W3C"`, but you're right that we should distinguish better.  I filed #278 on that, but if this lands first, we should probably do `"W3C"` here for consistency (and so it gets caught in the changes there).